record/record.app.js

```diff
@@ -125,6 +125,7 @@
                     }
 
                     model.column = col;
+                    model.baseTableName = $rootScope.reference.displayname.value;
```

Comment:
For table name, use `reference.table.name`. It's true that the reference displayname is often this value, but this may not be true in the future.

Reply:
I don't know the detail of the code, but want to state the objective which is to use the display value of the table that is shown on the page, so it makes sense to the user.

Reply:
I checked the different datatypes and found that for the base table the displayName and table.name are same but for related tables, the displayName and table.name might be different. So for the related entities, I have kept the displayName in the tooltip as that is what is displayed as the title as well.